examples/storybook/src/stories/index.ts

```diff
@@ -0,0 +1,568 @@
+import { storiesOf, moduleMetadata } from '@storybook/angular';
```

Comment:
Would be nice to spread them across multiple files. You can require them all at one from the config file with a blob pattern (see https://storybook.js.org/basics/writing-stories/#loading-stories-dynamically).

Reply:
done now! 

<img width="924" alt="screenshot at dec 03 18-43-11" src="https://user-images.githubusercontent.com/2982512/49391776-e8228c00-f72c-11e8-9376-e3cff14d5b04.png">